flare/gp_from_aimd.py

```diff
@@ -189,14 +195,30 @@ def run(self):
             self.pred_func(cur_frame, self.gp)
 
             # Convert to meV/A
-            mae = np.mean(np.abs(cur_frame.forces - dft_forces)) * 1000
+            error = np.abs(cur_frame.forces - dft_forces)
+            mae = np.mean(error) * 1000
+            mae_perspecies = {}
+            count_perspecies = {}
+            species = [Z_to_element(Z) for Z in set(cur_frame.coded_species)]
+            for ele in species:
+                mae_perspecies[ele] = 0
+                count_perspecies[ele] = 0
+            print(list(mae_perspecies.keys()))
```

Comment:
Was this a debug thing that got left in by mistake? If not, should add an if verbose: flag.

Reply:
oh yeah. forgot to remove this line...

Reply:
fixed in efd157402d391ec9091bb5041bd8a6bddf753c22